### DISCUSSION TITLE
My benefits still haven't arrived

### DISCUSSION THREAD

**Discussion by JairHan** [community/community#133555] [GitHub Education]
### Hi everyone,

My benefits still haven't arrived，He's been over five days

**Comment by Javokhdev** [comment]
how can we help you?

**Reply by JairHan** [reply]
Hi, h1096224896! You were last verified as a student on Jul 22, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.，But I haven't received it so far

**Comment by husnain112211** [comment]
What kind of benifit you are actually talking about? Can I help you? It would be a pleasure to me for sure..[.](https://beachbuggyracingapk.net/)..

**Reply by JairHan** [reply]
Student certification benefits，have not arrived yet

**Comment by JairHan** [comment]
Who can tell me that after applying for student certification and passing, it takes a few days to get student benefits? It passed on the 22nd. It's not working yet

**Comment by SohanKafle** [comment]
If you're talking about student benefits, It will take few days for the verification of your documents so that you'll be able to access the benifits don't worry it will come to you if all the documents you provided are valid and true.

**Comment by davevad93** [comment]
Hi @h1096224896 ,

What's your application status?

- `Congrats! Your request for @h1096224896 was approved and your benefits will be available soon.`

Or
 
- `Congrats! Your request for @h1096224896 was approved and your benefits are now available.` 

If it's the first one, you'll need to wait until your benefits are available, which may take even a couple of weeks during periods when there's a high volume of applications. If it's the latter, you should have full access to your benefits, in that case I suggest you to contact support by opening a ticket [here](https://support.github.com/contact/education), select `I'm a verified student and I'm having trouble redeeming my Global Campus offers.` and explain your issue.
